src/code/z_scene.c

```diff
@@ -3,116 +3,126 @@
 
 RomFile sNaviMsgFiles[];
 
-s32 Object_Spawn(ObjectContext* objectCtx, s16 objectId) {
+// called by
+// - initcontext: for gkeep
+// - spawn list scene command: for link's object
+// - special files scene command: for subkeep object if any
+s32 Object_SpawnPersistent(ObjectContext* objectCtx, s16 objectId) {
```

Comment:
I dont think its important to note what calls it
However I think the danger of using this function should be pointed out. I didnt realize until now the way that unk_09 (`numPersistentEntries`) works.
The wording can be worked on if needed, but these are the important points i think
```
Spawn an object file of the specified ID that will persist through room changes.

NOTE: This function is NOT meant to be called externally to spawn object files on the fly.
When an object is spawned with this function, any objects that come before it in the entry list will be treated as persistent. 
This function is only meant to be called internally by the object system BEFORE the object list from the room is processed.
```

Reply:
also, we should decide on persistent vs permanent. Actors use permanent to describe the same thing for their overlay allocation type. itd be good if they both used the same